src/ui/application-window.vala

```diff
@@ -89,6 +95,8 @@ namespace Peek.Ui {
         leave_recording_state ();
       });
 
+
+
```

Comment:
These blank lines were not there before so I think they should be removed. Same below.